twilio/http/http_client.py

```diff
@@ -40,11 +43,15 @@ def request(self, method, url, params=None, data=None, headers=None, auth=None,
 
         request = Request(method.upper(), url, params=params, data=data, headers=headers, auth=auth)
 
+        self.last_request = request
```

Comment:
Shouldn't these be `twilio.Request` not `requests.models.Request`? Same with the response object.

Reply:
yes thats what the other libraries do.